src/client.js

```diff
@@ -92,6 +95,14 @@ function createMethod(clientMethod, dbg, cancelCache) {
           call.removeListener('error', onError);
         };
 
+        // const originalUnsub = observer.unsubscribe;
```

Comment:
This is the key fix. With it un-commented calling unsubscribe either directly or via take() will destroy the stream, preventing the server from continuing to spam responses.